src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/tabs/fullScript_approved.properties

```diff
@@ -25,7 +19,8 @@ lastKnownApproverLogin=Last approver
 
 ## table content
 noRequesterSinceMetadata=N/A
-noRequesterSinceMetadata_tooltip=The requester was not recorded. It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
+noRequesterSinceMetadata_tooltip=The requester was not recorded. \
+  It could be due to the request being created before the metadata introduction or the code calling this script did not provide user information.
```

Comment:
```suggestion
  It could be because the request was created before the metadata was introduced or the code calling this script did not provide user information.
```